ansible_base/lib/utils/response.py

```diff
@@ -46,13 +47,19 @@ def get_fully_qualified_url(view_name: str, *args, **kwargs) -> str:
     Returns a fully qualified URL from the setting FRONT_END_URL or a DRF reverse if the setting is undefined
     NOTE: a DRF reverse could be relative or not depending on the request coming in
     '''
-    url = drf_reverse(view_name, *args, **kwargs)
     front_end_url = get_setting('FRONT_END_URL', None)
-    if not front_end_url:
-        return url
-
-    url_pieces = urlparse(url)
-    return f"{front_end_url.rstrip('/')}{url_pieces.path}"
+    if front_end_url:
+        url = drf_reverse(view_name, *args, **kwargs)
+        url_pieces = urlparse(url)
+        return f"{front_end_url.rstrip('/')}{url_pieces.path}"
+
+    if 'request' in kwargs and kwargs['request']:
+        request = kwargs.pop('request')
+    else:
+        request = get_current_request()
+
+    kwargs['request'] = request
```

Comment:
Non-blocking, could you just make this a `not` condition?
```suggestion
    if not kwargs.get('request', None):
        kwargs['request'] = get_current_request()
```